interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is the technology that GitHub, a code-sharing platform, uses. Git technology uses terminal to move code around. 
 
-Researched answer:
+Researched answer: Git is a DevOps tool used for source code management. It is a free and open-source version control system used to handle small to very large projects efficiently. Git is used to tracking changes in the source code, enabling multiple developers to work together on non-linear development. GitHub uses Git technology which is an online software development platform.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: An example of a JavaScript operator that will return a Boolean value is math problems with loose equality symbols.
 
-Researched answer:
+Researched answer: JavaScript operators that will return a Boolean value are those that use logical operators like AND and OR. 
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index is the locatiion of the variable in  an array. This starts at 0 and goes on infinitely. The difference between an index and a value is that a value is the numeric data that can mean anything. 
 
-Researched answer:
+Researched answer:An index is the locatiion of the variable in an array going from left to right. This can be either a number or letter and starts at 0 and goes on infinitely. The difference between an index and a value is that a value is a data chunk that represent pieces of information. 
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: An array is a list of variables such as names or numbers. A string is a one or more elements enclosed in quotation marks. They are both forms of data and can be manipulated using code. 
 
-Researched answer:
+Researched answer: An array used to store multiple values in a single variable while a string is a variable that consists of one or more letters, numbers, or symbols. They are similar because they contain data that can be manipulated by code. 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-Your answer:
+Your answer: Pair programming is a technique in which two coders are working on the same coding project. There is the "driver" who is the one typing out the code in accordance to the "navigator" who is deciding on the final code. Both coders are collaborating with eachout though on ideas for the code. 
 
-Researched answer:
+Researched answer: Pair programming is an agile software development technique in which two programmers work together at one workstation. The driver writes code while the other, the navigator reviews each line of code as it is typed in. The two programmers switch roles frequently.
```

Comment:
I can tell you put in a lot of effort on these answers - remember that the researched answers must be in your own words.  Please redo the researched answers and resubmit.